iree/compiler/Dialect/VM/Target/C/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+if(${IREE_ENABLE_EMITC})
+  iree_add_all_subdirs()
+  
+  iree_cc_library(
+    NAME
+      C
```

Comment:
Single letter directory and library name seems odd to me. Might be fine though?

Reply:
This follows the naming of the bytecode target

https://github.com/google/iree/blob/93e64ee7cf419ef068932d5261efc00e5771f375/iree/compiler/Dialect/VM/Target/Bytecode/CMakeLists.txt#L17-L19

However, we can rename to `CTarget` or `TargetC`, whereas the latter is more consistent with the naming used in MLIR (`MLIRTargetCpp`, `MLIRTargetAVX512`, ...).

Reply:
@ScottTodd Any further suggestion? :)

Reply:
Sticking with IREE's conventions works. Here are some other ideas for the directory / target name that could give more context:

* `Target/EmitC`
* `Target/Cpp`
* `Target/CCode`
* `Target/TextualC`
* `Target/CLang` (language... conflicts with https://clang.llvm.org/ though :P)
* `Target/StandaloneC`

I'm not really drawn to any of those though. `Target/C` SGTM.

<sub>_Naming things is hard_</sub>

Reply:
Okay, let's stick with `Target/C` for now, but I keep the other ideas in mind. Maybe we'll refactor later.